plugins/modules/ndo_port_channel_interface.py

```diff
@@ -0,0 +1,350 @@
+                    if interface_policy == policy_group["spec"]["name"]:
+                        interface_policy_uuid = policy_group["spec"]["uuid"]
+                        break
+                if not interface_policy_uuid:
+                    mso.fail_json(msg="Port Channel policy '{0}' not found in the list of existing Port Channel policy groups".format(interface_policy))
+            else:
+                mso.fail_json(msg="No existing Port Channel policy")
+
+        if match:
+            if match.details.get("name") != port_channel_interface:
+                ops.append(
+                    dict(
+                        op="replace",
+                        path="{0}/{1}/name".format(path, match.index),
+                        value=port_channel_interface,
+                    )
+                )
+                match.details["name"] = port_channel_interface
+
+            if match.details.get("description") != description:
+                ops.append(
+                    dict(
+                        op="replace",
+                        path="{0}/{1}/description".format(path, match.index),
+                        value=description,
+                    )
+                )
+                match.details["description"] = description
+
+            if match.details.get("node") != node:
+                ops.append(
+                    dict(
+                        op="replace",
+                        path="{0}/{1}/node".format(path, match.index),
+                        value=node,
+                    )
+                )
+                match.details["node"] = node
+
+            if match.details.get("memberInterfaces") != member_interfaces:
+                ops.append(
+                    dict(
+                        op="replace",
+                        path="{0}/{1}/memberInterfaces".format(path, match.index),
+                        value=",".join(member_interfaces),
+                    )
+                )
+                match.details["memberInterfaces"] = member_interfaces
+
+            if match.details.get("policy") != interface_policy_uuid:
+                ops.append(
+                    dict(
+                        op="replace",
+                        path="{0}/{1}/policy".format(path, match.index),
+                        value=interface_policy_uuid,
+                    )
+                )
+                match.details["policy"] = interface_policy_uuid
+
+            if interface_descriptions is not None and match.details.get("interfaceDescriptions") != interface_descriptions:
+                ops.append(
+                    dict(
+                        op="replace",
+                        path="{0}/{1}/interfaceDescriptions".format(path, match.index),
+                        value=[
+                            {
+                                "nodeID": node,
+                                "interfaceID": interface.get("interface_id"),
```

Comment:
do we allow settings description for interfaces that we have not defined as member, because UI does not allow you to do so

same applies for create flow

Reply:
answered above for the description.
but what do you mean by flow?